hermes-mock/src/main/java/pl/allegro/tech/hermes/mock/HermesMock.java

```diff
@@ -0,0 +1,147 @@
+package pl.allegro.tech.hermes.mock;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder;
+import com.github.tomakehurst.wiremock.verification.LoggedRequest;
+import org.junit.rules.MethodRule;
+import org.junit.runners.model.FrameworkMethod;
+import org.junit.runners.model.Statement;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.function.Function;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathMatching;
+import static com.jayway.awaitility.Awaitility.await;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static java.util.stream.Collectors.toList;
+
+public class HermesMock implements MethodRule {
```

Comment:
If I don't use JUnit I'd appreciate an implementation without these dependencies. I think it's useful to have a proxy to `HermesMock` which implements MethodRule, but it's not the essence in my opinion. A submodule with this proxy could satisfy the need for a JUnit rule, but we could reach a wider audience with a basic `HermesMock` that doesn't comply with `MethodRule` interface.